src/main/java/yoyo/command/CommandDeadline.java

```diff
@@ -0,0 +1,43 @@
+package yoyo.command;
+
+import java.time.LocalDateTime;
+
+import yoyo.core.DialogHandler;
+import yoyo.core.Parser;
+import yoyo.core.Storage;
+import yoyo.exception.YoyoException;
+import yoyo.task.Deadline;
+import yoyo.task.Task;
+import yoyo.task.TaskList;
+
+public class CommandDeadline extends Command {
+    public CommandDeadline(String[] inputTokens) {
+        super(inputTokens);
+    }
+
+    /**
+     * Executes "deadline" command.
+     *
+     * @param tasks Tasks currently in the Yoyo program.
+     * @param storage Storage instance of the Yoyo program.
+     * @param dialogHandler Ui instance of Yoyo program.
+     * @throws YoyoException
+     * @return
```

Comment:
There is a missing JavaDoc here for return value, and I noticed there were a few more similar cases in the other Command sub-classes.